imbi/opensearch/project.py

```diff
@@ -146,7 +146,7 @@ def _project_to_dict(value: models.Project) -> dict:
 class RequestHandlerMixin:
 
     def __init__(self, *args, **kwargs):
-        super(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
lol .... you should write a PEP requesting that this work 😮 
Since `super` no longer requires parameters, it would be nice if the call would automatically convert to `super().current_method()` through some deep magic.